src/api/callback.cc

```diff
@@ -158,20 +158,33 @@ MaybeLocal<Value> InternalMakeCallback(Environment* env,
     CHECK(!argv[i].IsEmpty());
 #endif
 
-  InternalCallbackScope scope(env, resource, asyncContext);
+  Local<Function> hook_cb = env->async_hooks_callback_trampoline();
+  int hook_count = 0;
+  int flags = 0;
```

Comment:
```suggestion
  int flags = InternalCallbackScope::kNoFlags;
```

Reply:
Thanks! I'll update that soon. Just running a benchmark now. :)